dumper/lib-ext-hive-metastore/build.gradle

```diff
@@ -78,15 +75,25 @@ shadowJar {
 
 }
 
-generateThriftSource {
-    generators {
-        java {
-            out file('build/generated-sources/thrift/main')
-            // option 'java5'
+task generateThrift {
```

Comment:
It's possible to say `def generateThrift = tasks.register('generateThrift') { ... }` instead of `task generateThrift { }` and the build will be slightly faster - only matters for large builds, because that does lazy configuration.